manifests/config.pp

```diff
@@ -59,6 +59,15 @@
       mode      => '0644',
       show_diff => $pulp::show_conf_diff,
     }
+
+    file { '/etc/pulp/server/plugins.conf.d/yum_distributor.json':
+      ensure    => file,
+      content   => template('pulp/yum_distributor.json.erb'),
```

Comment:
Other than a template, you can also use [to_pretty_json](https://forge.puppet.com/modules/puppetlabs/stdlib/reference#to_json_pretty).

Reply:
I like that! but i think i'm just gonna follow the existing convention here if thats okay